main.css

```diff
@@ -23,6 +23,42 @@ hr {
   background: linear-gradient(rgba(255, 193, 7, 0), #FF9800, rgba(255, 193, 7, 0));
 }
 
+.features {
+  background-color: #EEEEEE;
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.features::before{
+  background-image: url("img/triangle.png");
+  content: "";
+  width: 100%;
+  height: 10px;
+  background-repeat: repeat-x;
+  position: relative;
```

Comment:
set position here to absolute and on features parent set it to relative. That will position this element relatively to the first relative parent.